src/webui/webui.cpp

```diff
@@ -28,6 +28,10 @@
 
 #include "webui.h"
 
+#ifdef DISABLE_GUI
```

Comment:
In fact, we can not hope that this file will be included indirectly in GUI branch and include it explicitly in any case.

Reply:
you mean remove the `#if` conditional here?
but `QCoreApplication::exit(1);` below is only needed when using qbt-nox builds.

Reply:
Oh, yeah... Forget it.